src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -132,6 +136,9 @@ public RobotMap() {
 		Motor rightFlywheelMotor = new Motor(new VictorSP(Port.PWM.flywheelRightMotor));
 		rightFlywheelMotor.setInverted(false);
 		Component.flywheel = new Flywheel(leftFlywheelMotor, rightFlywheelMotor);
+		// Climber
+		Motor climbMotor = new Motor(new VictorSP(Port.PWM.climbMotor));
+		Component.climber = new Climber(climbMotor);
```

Comment:
Since Climber extends Motor, you don't need to wrap the Victor in a Motor object.
This should read:
```java
Component.climber = new Climber(new VictorSP(Port.PWM.climbMotor));
```
(or you're welcome to split it across two lines)